intro-to-R-tidyverse/00c-good-scientific-coding-practices.md

```diff
@@ -0,0 +1,51 @@
+## Tips for good scientific coding practices
+
+#### Style guides help people read your code
+
+Just like how incorrect punctuation and grammar can be distracting for a reader to grasp your message in your writing, code that doesn't follow a style is difficult for others to understand and be able to use.
+Your code is not useful if it isn't easily readable, which is why naming conventions, code style, and consistency are important.
+
+We suggest following a style guide like one of these:  
+- [Hadley Wickham's R Style Guide](http://adv-r.had.co.nz/Style.html)  
+- [Google's R Style Guide](https://google.github.io/styleguide/Rguide.xml).   
+
+#### `set.seed` helps people reproduce your results
+
+When performing any kind of analyses that use random sampling, or something that may vary each time you re-run it.
+
+*How to set the seed:*
+1) Put any number as your argument for the function and run `set.seed` like below.
+```
+set.seed(54321)
+```
+
+2) Run your analyses like normal.
+
+3) If you run your analyses again, and still have the seed set to the same number, you will get exactly the same results.
+```
+set.seed(54321)
+```
+
+Setting the seed makes it so what ever results you get, are reproducible *exactly*.
```

Comment:
```suggestion
Setting the seed makes so your results are *exactly* reproducible.
```
You might want to add a sentence that says that sometimes functions have a seed argument and/or acknowledge that whatever you are using to get those results need to respect `set.seed` for this sentence to hold true.